tstest/integration/vms/vms_test.go

```diff
@@ -459,6 +519,75 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 	}
 }
 
+func copyBinaries(t *testing.T, conn *ssh.Client) {
+	bins := integration.BuildTestBinaries(t)
+
+	cli, err := sftp.NewClient(conn)
+	if err != nil {
+		t.Fatalf("can't connect over sftp to copy binaries: %v", err)
+	}
+
+	mkdir(t, cli, "/usr/bin")
+	mkdir(t, cli, "/usr/sbin")
+	mkdir(t, cli, "/etc/systemd/system")
+	mkdir(t, cli, "/etc/default")
+
+	copyFile(t, cli, bins.Daemon, "/usr/sbin/tailscaled")
+	copyFile(t, cli, bins.Cli, "/usr/bin/tailscale")
+	copyFile(t, cli, "../../../cmd/tailscaled/tailscaled.defaults", "/etc/default/tailscaled")
+	copyFile(t, cli, "../../../cmd/tailscaled/tailscaled.service", "/etc/systemd/system/tailscaled.service")
+
+	t.Log("tailscale installed!")
+}
+
+func mkdir(t *testing.T, cli *sftp.Client, name string) {
+	t.Helper()
+
+	err := cli.MkdirAll(name)
+	if err != nil {
+		t.Fatalf("can't make %s: %v", name, err)
+	}
+}
+
+func copyFile(t *testing.T, cli *sftp.Client, fname, dest string) {
```

Comment:
localSrc, remoteDst?

then you don't need a comment. Otherwise it's not obvious and you'd want a comment.